utils/describe-ref/lib/describe-ref.js

```diff
@@ -24,11 +24,18 @@ function getArgs(options) {
     args.push("--match", options.match);
   }
 
+  if (commandOptions.includeMergedTags) {
+    // we want to consider all tags, also from merged branches
+    args = args.filter(arg => arg !== "--first-parent");
+    // also include lightweight tags
+    args.push("--tags");
```

Comment:
Did you consider these regarding lightweight tags?

* https://github.com/lerna/lerna/blob/aa4ae510b9c0a0935f0b1989859ba873a1e864cc/CHANGELOG.md#breaking-changes-2
* https://github.com/lerna/lerna/blob/f674f354f0260c57d885c181e16c9ce23ac252a8/doc/troubleshooting.md#publish-does-not-detect-manually-created-tags-in-fixed-mode-with-githubgithub-enterprise

Reply:
This doesn't really work for our case. As explained we want to tag in a release branch that gets merged into master. But the tagged commit is on the branch.

Reply:
I only want to point to the fact that lightweight tags aren't coompatible with lerna and it seems that it will hard to land this.

Reply:
I see but it isn't really just a lightweight tag, right?

Reply:
This is not going to fly. Lightweight tags are not intended to be used for tagging releases.